catalog/task/update-infra-deployments/0.1/update-infra-deployments.yaml

```diff
@@ -0,0 +1,332 @@
+    - name: scriptImage
+      description: Image reference for SCRIPT execution
+      default: >-
+        quay.io/hacbs-release/release-utils@sha256:5298e31e7e7a97cab005750096abe8dbfca3f422cf049fd87de76a117072a9b5
+    - name: sharedSecret
+      default: infra-deployments-pr-creator
+      description: secret in the namespace which contains private key for the GitHub App
+    - name: githubAppID
+      description: ID of Github app used for updating PR
+      default: "305606"
+    - name: githubAppInstallationID
+      description: Installation ID of Github app in the organization
+      default: "35269675"
+  volumes:
+    - name: infra-deployments-pr-creator
+      secret:
+        # 'private-key' - private key for Github app
+        secretName: $(params.sharedSecret)
+    - name: shared-dir
+      emptyDir: {}
+
+  steps:
+    - name: git-clone-infra-deployments
+      image: $(params.gitImage)
+      volumeMounts:
+        - name: shared-dir
+          mountPath: /shared
+      workingDir: /shared
+      env:
+        - name: TARGET_GH_REPO
+          value: "$(params.targetGHRepo)"
+      script: |
+        git clone "https://github.com/${TARGET_GH_REPO}.git" cloned
+    - name: run-update-script
+      image: $(params.scriptImage)
+      volumeMounts:
+        - name: shared-dir
+          mountPath: /shared
+      workingDir: /shared
+      script: |
+        cd cloned
+        
+        echo "snapshot:"
+        cat "$(params.snapshotPath)"
+        echo ""
+        
+        # we assume we only have one component in the service.
+        #
+        revision=$(cat "$(params.snapshotPath)" | jq -r .components[0].source.git.revision)
+        echo "revision: $revision"
+        echo $revision > ../revision.txt
+
+        originRepo=$(cat "$(params.snapshotPath)" | jq -r .components[0].source.git.url)
+        echo "origin repo: $originRepo"
+        echo $originRepo > ../originRepo.txt
+
+        # get SCRIPT from ExtraData
+        echo "extraData: $(params.extraDataJsonPath)"
+        SCRIPT=$(cat "$(params.extraDataJsonPath)" | jq -r '."infra-deployment-update-script"')
+
+        if [ "${SCRIPT}" == "" ] ; then
+          echo "No script provided via 'infra-deployment-update-script' key in extraData"
+          exit 0
+        fi
+
+        echo "original script:"
+        echo "$SCRIPT"
+        PATCHED_SCRIPT=$(echo "$SCRIPT" | sed "s/{{ revision }}/$revision/g")
+        echo "patched script:"
+        echo "$PATCHED_SCRIPT"
+        echo "$PATCHED_SCRIPT" | sh
+    - name: get-diff-files
+      image: $(params.gitImage)
+      volumeMounts:
+        - name: shared-dir
+          mountPath: /shared
+      workingDir: /shared
+      script: |
+        cd cloned
+        git status -s --porcelain | cut -c4- > ../updated_files.txt
+    # Based on https://github.com/tektoncd/catalog/tree/main/task/github-app-token/0.2/
+    - name: create-mr
```

Comment:
if it is github, it is a pr, not an mr, but I don't know that diverging from the upstream task for this is worth it

Reply:
fixed